### PR TITLE
Fixed error: "array key does not exist"

### PR DESCRIPTION
**Pull request by OzanKurt**

### CODE REVIEW

**Comment by OzanKurt** [comment]
@Adyg

File: lib/Imgur/Api/Model/Basic.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ class Basic {
     public function __construct($parameters) {
         $this->setData($parameters['data'])
              ->setSuccess($parameters['success'])
-             ->setStatus($parameters['status']);
+             ->setStatus(@$parameters['status']);
```
Comment (widnyana) [inline lib/Imgur/Api/Model/Basic.php]: I think this is not a proper solution :|

Comment (OzanKurt) [inline lib/Imgur/Api/Model/Basic.php]: It is not ;/

Well you might wanna do something like:

```php
if (array_key_exists('status', $parameters)) {
    $this->setStatus($parameters['status']);
}
```

**Comment by OzanKurt** [comment]
Please update.

**Comment by j0k3r** [comment]
Sorry @OzanKurt this isn't a good solution.

I'm currently working on rewriting the client and I plan to remove Model to fix that issue (and many other one). You can see the work in progress in the [`tests`](https://github.com/j0k3r/php-imgur-api-client/tree/tests) branch (which will be the next 2.0.0)

**Pull request closed without merging**